common/src/test/java/io/druid/common/guava/CombiningSequenceTest.java

```diff
@@ -271,4 +274,90 @@ public boolean apply(
     Assert.assertFalse(expectedVals.hasNext());
     yielder.close();
   }
+
+  @Test
+  public void testFails()
```

Comment:
can we give the test a proper name? This currently makes it seems we should expect this test to fail.